Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpBuildDeployPreferences.cs

```diff
@@ -28,6 +29,15 @@ public static string BuildConfig
             set => EditorPreferences.Set(EDITOR_PREF_BUILD_CONFIG, value.ToLower());
         }
 
+        /// <summary>
+        /// The current Build Configuration. (Debug, Release, or Master)
```

Comment:
Note to self: Update this when it goes in (since I don't have rights on the fork)